ocs_ci/ocs/ripsaw.py

```diff
@@ -124,6 +124,8 @@ def cleanup(self):
         run(f'oc delete -f {self.operator}', shell=True, cwd=self.dir)
         run(f'oc delete -f deploy', shell=True, cwd=self.dir)
         run_cmd(f'oc delete project {self.namespace}')
+        # Reset namespace to default 'openshift-storage'
+        run(f'oc project openshift-storage', shell=True)
```

Comment:
You have a function for that

Reply:
Can you clarify on this comment?  We don't have a function to revert namespace to default.

Reply:
openshift-storage shouldn't be hard coded but read from config.ENV_DATA['cluster_namespace']

Reply:
Fixed.  Used switch_to_default_rook_cluster_project() to revert default namespace.

Reply:
Yes @tiffanyn108 , that's the right function